HousingFinanceInterimApi/serverless.yml

```diff
@@ -109,7 +109,8 @@ functions:
         - schedule: cron(0 4 * * ? *)    
   refreshManageArrearsTables:
     name: ${self:service}-${self:provider.stage}-refresh-ma-cur-bal
-    description: "The scheduler to refresh manager arrears balance based on current balance table."
+    description: "The scheduler to refresh manager arrears balance based on current balance table."
```

Comment:
change manager arrears to manage arrears

Reply:
Fixed